### PR TITLE
Feature/delete project

### PR DESCRIPTION
**Pull request by SukranSari**

### CODE REVIEW

File: components/Modal/ModalContent.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+import { useRouter } from "next/router";
+
+export default function ModalContent({
+  onClose,
+  id,
+  newProjects,
+  setNewProjects,
+}) {
+  const router = useRouter();
```
Comment (NikRoe) [inline components/Modal/ModalContent.js:11]:
```suggestion
```

File: components/Modal/ModalContent.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
+import styled from "styled-components";
+
+import { useRouter } from "next/router";
```
Comment (NikRoe) [inline components/Modal/ModalContent.js:3]:
```suggestion
```

**Review by NikRoe — CHANGES_REQUESTED** [review]
Hi there,

a promising start!

I left some notes with the help of which you can improve the reusability of your modal and get rid of some props that you don't need.

Keep it up 🙌

File: pages/_app.js [hunk 3]
```diff
@@ -2,10 +2,12 @@ import GlobalStyle from "@/styles";
 import projects from "@/lib/projects.js";
 import { Saira } from "next/font/google";
 import useLocalStorageState from "use-local-storage-state";
-
 const saira = Saira({ subsets: ["latin"] });
```
[the remark this answers is not in the document]
Comment (eduskx) [reply pages/_app.js:6]: Thanks for the hint! We will consider this in a future user story.

**Pull request merged**